### PR TITLE
Cursor style- Grab

### LINKED ISSUE #4: Cursor design can be changed

### PR DESCRIPTION
**Pull request by JuanitaCathy**
The style of the cursor changes when hovered over. If you like it, please label it for the fest.

### CODE REVIEW

**Comment by eduardconstantin** [comment]
Thank you for opening a PR!
Could you change it to `pointer`, I think it makes more sense like this.

File: src/index.css [hunk 1]
```diff
@@ -26,4 +26,5 @@ code {
 	align-items: center;
 	background-size: 1%;
 	background-image: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23000000' fill-opacity='0.2' fill-rule='evenodd'%3E%3Ccircle cx='3' cy='3' r='3'/%3E%3Ccircle cx='13' cy='13' r='3'/%3E%3C/g%3E%3C/svg%3E");
+    cursor:grab;
```
Comment (eduardconstantin) [inline src/index.css]:
```suggestion
    cursor:pointer;
```

**Comment by JuanitaCathy** [comment]
Okay.

File: src/index.css (hunk 1 above)
Comment (JuanitaCathy) [inline src/index.css:29]: ty

**Comment by JuanitaCathy** [comment]
It says pull request authors cannot approve the change.

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by eduardconstantin** [comment]
Sorry I forgot to submit the review

**Comment by cyberpvn7** [comment]
Assign Me

**Comment by eduardconstantin** [comment]
> Assign Me 

This one is taken.

**Pull request merged**